plugins/domains/d.pr.js

```diff
@@ -23,12 +20,22 @@ module.exports = {
                 // height: oembed.height
             };
         }
+        if (/mp4/i.test(oembed.variant)) {
+            var url = cheerio('video source').attr('src');
```

Comment:
`url` is reserved for input URL. Better not to use it in plugins because it misleads easily. As you use if for video src, why not call it simply `videoSrc` ?